### PR TITLE
Storage release notes 14-02-2023 

### PR DESCRIPTION
**Pull request by shanyp**
Preview:
https://neon-next-git-fix-14-02-2023-release-neondatabase.vercel.app/docs/release-notes
https://neon-next-git-fix-14-02-2023-release-neondatabase.vercel.app/docs/reference/pg-extensions

### CODE REVIEW

**Review by vadim2404 — APPROVED** [review]

File: content/release-notes/2023-02-14.md [hunk 1]
```diff
@@ -5,7 +5,5 @@ label: 'Storage'
 ### Features
 
 - Compute: Added support for the the PostgreSQL `pgvector`, `plls` and `plcoffee` extensions. For more information about PostgreSQL extensions supported by Neon, see [PostgreSQL extensions](https://neon.tech/docs/reference/pg-extensions/).
-- Proxy: Latencies have been drastically decreased for websocket and pool connections.
-- Pageserver: Experimental layer eviction feature to reduce non used storage layers.
-- Pageserver: Documented the synthetic size calculation.
-
+- Proxy: Reduced network latencies for WebSocket and pooled connections to the Neon Proxy by enabling the `TCP_NODELAY` option. Enabling this option causes segments to be sent as soon as possible, even if there is only a small amount of data. For more information about the `TCP_NODELAY` option, refer to the [TCP protocol man page](https://linux.die.net/man/7/tcp).
```
Comment (vadim2404) [inline content/release-notes/2023-02-14.md:8]: It's not only one item. @funbringer introduced the cache for this

Comment (danieltprice) [reply content/release-notes/2023-02-14.md:8]: Hi @vadim2404 Is there a PR for that I can look it? The only thing regarding caching I see is this one, but it does not seem to be related: https://github.com/neondatabase/neon/pull/3605

Comment (vadim2404) [reply content/release-notes/2023-02-14.md:8]: You mentioned valid PR. but there was one more https://github.com/neondatabase/neon/pull/3331

**Review by danieltprice — APPROVED** [review]
lgtm

File: content/release-notes/2023-02-14.md (hunk 1 above)
Comment (danieltprice) [reply content/release-notes/2023-02-14.md]: Revised the entry to mention #3331

**Review by vadim2404 — APPROVED** [review]

**Pull request merged**